examples/language/lib/class_modifiers/ex3/a.dart

```diff
@@ -0,0 +1,6 @@
+// Library a.dart
+interface class Vehicle {
+  void moveForward(int meters) {
+    //...
```

Comment:
To be consistent with the other ellipsis comments:

```suggestion
    // ...
```

_ellipsis is always a hard word for me to spell for some reason??_

Reply:
haha!

Reply:
done!